packages/p2p/stores/base_store.js

```diff
@@ -0,0 +1,5 @@
+export default class BaseStore {
```

Comment:
What is `BaseStore` for?

Reply:
It saves us from writing the same code again and again. Most of our store constructors look like below:

```JS
constructor(root_store) {
    this.root_store = root_store;
}
```

By extending from `BaseStore` this automatically happens.

Reply:
Sounds cool, nicely done. 👏